components/modal.css

```diff
@@ -0,0 +1,45 @@
+.modal h1 {
+  font-size: 3rem;
+  font-weight: bold;
+}
+.demo-modal {
+  width: fit-content;
+  height: 10rem;
+  background-color: white;
+  border-radius: 0.5rem;
+  box-shadow: #00000059 0px 5px 15px;
```

Comment:
using color value directly is not good practice

Reply:
Thanks for the suggestion. I have done that changes. actually, I am using variables for elements that I have defined in other folders too. here I wanted to use new shades only for these components so that's why I used directly.